web/app/services/recently-viewed-docs.ts

```diff
@@ -308,3 +308,9 @@ export default class RecentlyViewedDocsService extends Service {
     }
   );
 }
+
+declare module "@ember/service" {
+  interface Registry {
+    "recently-viewed-docs": RecentlyViewedDocsService;
+  }
```

Comment:
This is to allow aliasing in TS files, e.g., `@service('recently-viewed-docs') declare recentDocs;`